.github/script/send-test-report-prev.js

```diff
@@ -0,0 +1,16 @@
+// Send test report is a minimal script that makes a HTTP POST call to a webhook to send a slack message.
+// For example, this script passes results of the browser compatibility report to the webhook so that the amazon-chime-js-sdk team can be notified of the results.
+
+const axios = require('axios');
+
+var myArgs = process.argv.slice(3);
+
+axios.post(myArgs[0], {
```

Comment:
Can you document why we cant reuse the other script here?

Reply:
I actually ended up just editing the webhook to take in the same params as normal browser compatibility, so this is no longer needed. Previously I added a extra variable for the version number, but since we're only going to be supporting one previous version it's not really needed, so now I just removed it so it will work fine, but we're still passing the version number as a variable so this PR will remove it